Src/Support/GoogleApis/Apis/[Media]/Download/MediaDownloader.cs

```diff
@@ -148,8 +153,12 @@ public IDownloadProgress Download(string url, Stream stream)
         #endregion
 
         /// <summary>
-        /// The core download logic. It downloads the media in parts, where each part's size is defined by 
-        /// <see cref="ChunkSize"/> (in bytes).
+        /// The core download logic. We download the media and write it to an output stream
+        /// ChunkSize bytes at a time, raising the ProgressChanged event after each chunk.
+        /// 
+        /// The chunking behavior is largely a historical artifact: a previous implementation
+        /// issued multiple web requests, each for ChunkSize bytes. Now we do everything in
+        /// one request, but the API and client-visible behavior are retained for compatibility.

```

Comment:
Yes, as before we buffer only up to ChunkSize bytes in memory before writing them to the client's stream.